src/components/Home.jsx

```diff
@@ -172,17 +172,19 @@ function Home() {
                                     className={`p-[1rem] text-[1.5rem] scale-[0.7] rounded-[50%] shadow-md hover:bg-gray-1000 transform transition duration-300 ease-in-out hover:md:scale-[0.8] cursor-pointer bg-black dark:bg-amber-400 shadow-black dark:shadow-black dark:shadow-md ${!showMenuAndBgColor && "mt-10"
                                         }`}
                                     onClick={toggleDarkMode}
+                                    
                                 >
                                     {darkMode ? (
-                                        <FaSun className="text-black" />
+                                        <FaSun className="text-black " title="LightMode" />
                                     ) : (
-                                        <FaMoon className="text-white" />
+                                        <FaMoon className="text-white" title="DarkMode"/>
                                     )}
```

Comment:
### Hi there @kekubhai 👋🏻

The changes you made to improve accessibility is really impressive, but you still need to refactor this part a little bit. So that we can make truly accessible Theme Toggle.

You need to change the `div` into a `button` element, Then you need to add `aria-label` with `"Theme Toggle - ${darkMode ? "Light" : "Dark"} Enabled"`. Now you can remove `title` from FontAwesome Icons, and add `title="Theme Toggler"` for the `button` element itself.

`aria-label` would help us in assisting accessibility devices, whereas `title` would provide a visual tooltip (only accessible for user who don't have visual problems).

---

Additionally, I've seen lot `title` attributes which are added to interactive elements. There's a thing you need to do, it's nothing much complicated. Just add `aria-label` attribute (for every elements where add you added `title`) copy the contents of `title` for `aria-label` so that we can assure users who have visual problems, they might rely on screen readers like accessibility devices.